back/spec/acceptance/project_folders_spec.rb

```diff
@@ -128,6 +128,30 @@
       end
     end
 
+    get 'web_api/v1/project_folders/for_admin' do
+      with_options scope: :page do
+        parameter :number, 'Page number'
+        parameter :size, 'Number of folders per page'
+      end
+
+      example_request 'List all folders for admin' do
+        assert_status 200
+        json_response = json_parse(response_body)
+        expect(json_response[:data].size).to eq 2
```

Comment:
Helper methods can make this more concise:
```suggestion
        expect(response_data.size).to eq 2
```